### PR TITLE
finished the layout

### PR DESCRIPTION
**Pull request by LailaL88**

### CODE REVIEW

File: wp-content/themes/poetiktheme/dist/app.css [hunk 1]
```diff
@@ -22,28 +22,549 @@ body {
...
 h1,
-h2,
-h3 {
+h2 {
   font-family: "Rammetto One", cursive;
 }
 
-.image__slide img {
-  width: 100%;
-  height: auto;
```
Comment (srlhyo) [inline wp-content/themes/poetiktheme/dist/app.css:44]: Is it any particular reason why you got rid of the rule?

File: wp-content/themes/poetiktheme/front-page.php [hunk 2]
```diff
@@ -10,17 +10,22 @@
...
-        <ul class="social-icons">
-            <li><a href="#"><i class="fab fa-facebook-f"></i></a></li>
-            <li><a href="#"><i class="fab fa-twitter"></i></a></li>
-            <li><a href="#"><i class="fab fa-instagram"></i></a></li>
-            <li><a href="#"><i class="fab fa-soundcloud"></i></a></li>
-        </ul>
+    <header class="header">
+        <div class="header__top">
+            <div class="header__logo-wrapper">
+            <img class="header__logo" src="<?php echo get_stylesheet_directory_uri(); ?>./src/images/POETIK.png" alt="logo">
```
Comment (srlhyo) [inline wp-content/themes/poetiktheme/front-page.php:16]: Well done! I also think this way is better performance wise. Too many google fonts isn't a good a idea.

File: wp-content/themes/poetiktheme/front-page.php [hunk 3]
```diff
@@ -10,17 +10,22 @@
...
+            </div>
+            <!-- <h1 class="logo">poetik</h1> -->
```
Comment (srlhyo) [inline wp-content/themes/poetiktheme/front-page.php]: Is this needed? If not, I'd suggest you should get rid of it to keep the code base as clean as possible right from the start.

File: wp-content/themes/poetiktheme/front-page.php [hunk 4]
```diff
@@ -324,175 +331,182 @@
...
+                </div>
+                <div class="photo" style="background-image: url('<?php echo get_stylesheet_directory_uri(); ?>./src/images/bridge.jpeg')">
+                    <!-- <img src="<?php echo get_stylesheet_directory_uri(); ?>./src/images/bridge.jpeg" alt="bridge"> -->
+                </div>
+                <div class="photo" style="background-image: url('<?php echo get_stylesheet_directory_uri(); ?>./src/images/live.jpeg')">
+                    <!-- <img src="<?php echo get_stylesheet_directory_uri(); ?>./src/images/live.jpeg" alt="live"> -->
+                </div>
+                <div class="photo" style="background-image: url('<?php echo get_stylesheet_directory_uri(); ?>./src/images/staring.jpeg')">
+                    <!-- <img src="<?php echo get_stylesheet_directory_uri(); ?>./src/images/staring.jpeg" alt="staring"> -->
+                </div>
```
Comment (srlhyo) [inline wp-content/themes/poetiktheme/front-page.php]: Same here. Do we need those comments?

File: wp-content/themes/poetiktheme/front-page.php [hunk 5]
```diff
@@ -324,175 +331,182 @@
...
-                        <img src="" alt="">
-                    </div>
+    <section class="shop">
+        <div class="wrapper">
+            <h2 class="title">shop</h2>
+            <div class="shop__small-screen">
+                <div class="shop__slider">
+                </div>
+                <div class="shop__item-price">
+                    <p>price: 21.89</p>
```
Comment (srlhyo) [inline wp-content/themes/poetiktheme/front-page.php:371]: Here is only a suggestion:  we could use an character entity for the currency, perhaps. Anyway, it's minor! Let's just have that in mind for later :)

File: wp-content/themes/poetiktheme/front-page.php [hunk 6]
```diff
@@ -324,175 +331,182 @@
...
+                </div>
 
-                    <div class="shop__item-message">
-                        <p>sold out</p>
+                <button class="btn btn__buy">BUY</button>
```
Comment (srlhyo) [inline wp-content/themes/poetiktheme/front-page.php]: Hummm... take a quick look at this document :  [https://medium.com/@mandy.michael/why-you-shouldnt-write-your-content-in-uppercase-instead-use-css-b03ac2c65b99#:~:text=Responses%20(1)-,Why%20you%20shouldn't%20write%20your,in%20Uppercase%2C%20instead%20use%20CSS.&text=With%20that%20in%20mind%2C%20if,the%20text%2Dtransform%20CSS%20property.](url)

**Review by srlhyo — COMMENTED** [review]
Please take a look at all the suggestion and address them accordingly.

File: wp-content/themes/poetiktheme/dist/app.css (hunk 1 above)
Comment (LailaL88) [reply wp-content/themes/poetiktheme/dist/app.css:44]: I dont get where was this initially! What is .image__slide? I think I dont even have such class any more...

File: wp-content/themes/poetiktheme/front-page.php (hunk 2 above)
Comment (LailaL88) [reply wp-content/themes/poetiktheme/front-page.php:16]: :)

File: wp-content/themes/poetiktheme/front-page.php (hunk 3 above)
Comment (LailaL88) [reply wp-content/themes/poetiktheme/front-page.php]: No this comment is not needed, I just forgot to delete...

File: wp-content/themes/poetiktheme/front-page.php (hunk 4 above)
Comment (LailaL88) [reply wp-content/themes/poetiktheme/front-page.php]: No, dont need comments

File: wp-content/themes/poetiktheme/front-page.php (hunk 5 above)
Comment (LailaL88) [reply wp-content/themes/poetiktheme/front-page.php:371]: Yeah could add that)

File: wp-content/themes/poetiktheme/front-page.php (hunk 6 above)
Comment (LailaL88) [reply wp-content/themes/poetiktheme/front-page.php]: It gives 404 page....but yeah I will change it to lower case

File: wp-content/themes/poetiktheme/dist/app.css (hunk 1 above)
Comment (srlhyo) [reply wp-content/themes/poetiktheme/dist/app.css:44]: ahahah this rules comes from the splide library (for the slider).

**Review by srlhyo — APPROVED** [review]
All comments address! Branch will be merged and deleted.

**Pull request merged**